sdk/azcore/runtime/request_test.go

```diff
@@ -341,6 +342,10 @@ func TestCloneWithoutReadOnlyFieldsEndToEnd(t *testing.T) {
 		ID:   &id,
 		Name: &name,
 	}
+
+	os.Setenv("AZURE_SDK_GO_OMIT_READONLY", "true")
+	defer os.Unsetenv("AZURE_SDK_GO_OMIT_READONLY")
```

Comment:
```suggestion
	t.Setenv("AZURE_SDK_GO_OMIT_READONLY", "true")
```

Reply:
That's awesome, I didn't know about this feature.